_sass/blocks/explore/_overview.scss

```diff
@@ -1,11 +1,25 @@
+.explore-section_top,
 .explore-section {
-	border-bottom: $base-border-size solid $dark-gray;
-	margin-bottom: $base-padding;
+	padding-bottom: $base-padding;
+	padding-top: $base-padding;	
+}
+
+.explore-section {
+	border-bottom: $thin-border-size solid $mid-gray;
+}
+
+.explore-section_category {
+  margin-top: $base-padding;
```

Comment:
Properties should be ordered border-bottom, margin-bottom, margin-top, padding-bottom